### PR TITLE
docs: add contributing guide

### PR DESCRIPTION
**Pull request by UlisesGascon**
This is an initial version, feel free to add your feedback 👍

### CODE REVIEW

**Comment by UlisesGascon** [comment]
This is ready for review (cc: @expressjs/express-tc)

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+We are open to, and grateful for, any contributions made by the community. By participating in this project, you agree to abide by [Express.js Code of Conduct](https://github.com/expressjs/express/blob/master/Code-Of-Conduct.md).
+
+## How to Contribute
+
+The Security Working Group is open to anyone who is interested in security and wants to contribute to the security of the Express.js ecosystem. You don't need to be a security expert to join the group, but you need to be passionate about security and willing to learn and contribute.
+
+The participation in the Security Working Group is open to everyone, and we encourage you to join the group and contribute in the following ways:
```
Comment (marco-ippolito) [inline Contributing.md:9]:
```suggestion
The participation in the Security Working Group is open to everyone, and we encourage you to join the group and contribute in the following ways:
```

it sounds like a repetition from the lines above "its open to everyone"

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Participate in the meetings
+- Participate in the offline discussions
+- Contribute to the [GitHub issues](https://github.com/expressjs/security-wg/issues)
+- Provide feedback on the security policies and procedures
+- Contribute to the security guidelines and recommendations
+
+The group is composed by two groups of members: the Security Triage Team and the Regular members. The regular members are responsible for the public facing activity of the group, while the Security Triage Team is responsible for the security triage process.
+
+You don't need to be a member of the Security Working Group to contribute to the security of the Express.js ecosystem.
```
Comment (marco-ippolito) [inline Contributing.md:18]:
```suggestion
You don't need to be a member of the Security Working Group to contribute to the security of the Express.js ecosystem.
```
this also feels reduntant

Comment (UlisesGascon) [reply Contributing.md]: I removed this

File: Contributing.md (hunk 1 above)
Comment (UlisesGascon) [reply Contributing.md]: I rewritten a bit the paragraph

**Review by inigomarquinez — COMMENTED** [review]
LGTM!

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- Provide feedback on the security policies and procedures
+- Contribute to the security guidelines and recommendations
+
+The group is composed by two groups of members: the Security Triage Team and the Regular members. The regular members are responsible for the public facing activity of the group, while the Security Triage Team is responsible for the security triage process.
+
+## How to join the Security Working Group?
+
+To join the Security Working Group, you need to:
+- Join the [OpenJS Foundation Slack](https://openjsf.org/collaboration)
+- Join the channel `#express-security` in the [OpenJS Foundation Slack](https://openjsf.org/collaboration)
```
Comment (wesleytodd) [inline Contributing.md:20]: Do we think we need a new channel for this? I think it is easier to start consolidated in #express and then expand to a security one later if necessary than to reverse the decision (as that requires archiving a channel).

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- Contribute to the group activities
+- Discuss with the existing members about your interest in joining the Security Triage Team, so they can evaluate your participation and invite you to join the team.
+
+
+## How to join the Security Triage Team?
+
+By default, all the Technical Committee (TC) members, and repo captains are part of the Security Triage Team. Also, the Security Triage Team can invite other members to join the team based on their participation and contributions to the group activities.
+
+It is expected that the Security Triage Team members have a good understanding of the Express.js ecosystem and the security practices, and are willing to contribute to the security of the Express.js ecosystem in a long term.
```
Comment (wesleytodd) [inline Contributing.md:29]:
```suggestion
It is expected that the Security Triage Team members have a good understanding of the Express ecosystem and the security practices, and are willing to contribute to the security of the Express ecosystem in a long term.
```

**Review by wesleytodd — APPROVED** [review]
Other than my two very small nitpick's LGTM!

**Comment by crandmck** [comment]
I second the two comments from @wesleytodd 
Otherwise, LGTM

File: Contributing.md (hunk 3 above)
Comment (UlisesGascon) [reply Contributing.md:20]: I actually created the channel, but I agree that the segmentation currently is not necessary. I will update it

Comment (UlisesGascon) [reply Contributing.md]: landed in [dcb40d6](https://github.com/expressjs/security-wg/pull/4/commits/dcb40d6513a3068d69dc1be34b2de3e846f35890)

**Review by inigomarquinez — APPROVED** [review]
LGTM!

**Comment by UlisesGascon** [comment]
I will merge the PR as the suggestions were addressed 👍

**Pull request merged**